.github/workflows/TestCloudFunction.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Cloud Functions
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
```

Comment:
You don't need to include the `pull_request` option if you already have `push` enabled. 

There are cases where both are necessary. E.g.,when triggering on pushes/PRs on specific branches. However, this is not one of those cases.